style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
- [ OPTIONAL ] From the project requirement, your CSS file should include one class with styles for your header, meaning you should use a CSS class selector to style your heading.

Reply:
Okay, but why is mine not being accepted while it's the same as what my partners did?